nano/lib/tomlconfig.cpp

```diff
@@ -186,25 +186,24 @@ void nano::tomlconfig::erase_default_values (tomlconfig & defaults_a)
 	erase_defaults (defaults_l.get_tree (), self.get_tree (), get_tree ());
 }
 
-std::string nano::tomlconfig::to_string ()
-{
-	std::stringstream ss;
-	cpptoml::toml_writer writer{ ss, "" };
-	tree->accept (writer);
-	return ss.str ();
-}
-
-std::string nano::tomlconfig::to_string_commented_entries ()
+std::string nano::tomlconfig::to_string (bool comment_values)
 {
 	std::stringstream ss, ss_processed;
 	cpptoml::toml_writer writer{ ss, "" };
```

Comment:
Looking at the toml_writer constructor it looks like there is already support for indenting that we're overriding by passing in an empty string for the second argument.

`    /**
     * Construct a toml_writer that will write to the given stream
     */
    toml_writer(std::ostream& s, const std::string& indent_space = "\t")`
    
    I believe we should be using that rather than implementing our own.

Reply:
Unfortunately the build in indent function only indents the non-commented lines. Looks like this:
```
[node.rep_crawler]
                query_timeout = 60000

[node.rocksdb]

# Whether to use the RocksDB backend for the ledger database.
# type:bool
                enable = false

# Number of threads to use with the background compaction and flushing. Number of hardware threads is recommended.
# type:uint32
                io_threads = 20

# This will modify how much memory is used represented by 1 (low), 2 (medium), 3 (high). Default is 2.
# type:uint8
                memory_multiplier = 2

[node.statistics.log]

# Log file name for counters.
# type:string
                        filename_counters = "counters.stat"
```